spec/robots/dor_repo/accession/reset_workspace_spec.rb

```diff
@@ -7,20 +7,20 @@
   let(:robot) { described_class.new }
   let(:object_client) { instance_double(Dor::Services::Client::Object, workspace: workspace_client) }
   let(:workspace_client) { instance_double(Dor::Services::Client::Workspace, reset: nil) }
+  let(:process) { instance_double(Dor::Workflow::Response::Process, lane_id: 'default') }
+  let(:workflow_client) { instance_double(Dor::Workflow::Client, process:) }
 
   before do
     allow(Dor::Services::Client).to receive(:object).with(druid).and_return(object_client)
+    allow(LyberCore::WorkflowClientFactory).to receive(:build).and_return(workflow_client)
   end
 
   describe '#perform' do
     subject(:perform) { test_perform(robot, druid) }
 
-    before do
-      perform
-    end
-
```

Comment:
without this ... how is this testing what happens when the reset step is performed?

Reply:
looks like it now gets called inline in the test.  i guess the other option would've been to replace the `before` with something like `let(:status) { perform }` and then `expect` on `status` instead of `perform.status`.  i might like what's here slightly better than that, but i think those options are similarly clear.

Reply:
Refactored to make this a tad clearer.